SConstruct

```diff
@@ -1,19 +1,26 @@
 # -*- python -*-
 
+# Importing "lsst.sconsUtils" will cause Python
+# to load the __init__.py which will cause lsstimport to
+# be loaded which will hopefully not fail with a fatal error
+# but will initially print out a warning that base has not
+# been built yet.
```

Comment:
The commit message says that this "will load and fail first time". The above says it will "hopefully not fail". Reading those, I'm a bit unclear what's actually going to happen, how deterministic it is, and what the consequences will be. Could you maybe reword a bit for clarity?